x-pack/plugins/maps/public/actions/ui_actions.ts

```diff
@@ -123,3 +124,15 @@ export function closeTimeslider() {
     dispatch(setQuery({ clearTimeslice: true }));
   };
 }
+
+export function openLayerWizard(autoOpenLayerWizardId: string) {
```

Comment:
Can you also rename the method to something more inline with the state its updating. Something like `setAutoOpenLayerWizardId`?